otterdog/eclipse-cdt-cloud.jsonnet

```diff
@@ -314,6 +314,7 @@ orgs.newOrg('eclipse-cdt-cloud') {
       },
     },
     orgs.newRepo('vscode-svd-viewer') {
+      aliases: ['vscode-peripheral-inspector'],
```

Comment:
it should the other way around, you change the name of the repo to the new name, and add an aliases property capturing the previous name so that we can associate the existing repo with the new name.